src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -518,6 +518,11 @@ In this message, the term "binding" means a piece of generated code that makes i
   <data name="XA3007" xml:space="preserve">
     <value>Could not link native shared library: {0}</value>
   </data>
+  <data name="XA3008" xml:space="preserve">
+    <value>Unable to strip native library: {0}, strip tool exited with code: {1}</value>
+    <comment>{0} - the native library path
+{1} - integer number exit code</comment>
+  </data>
```

Comment:
Oh, maybe we don't need this error message anymore?

Reply:
Removed